File: vs2003_cd02/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/QuickStart/winforms/samples/data/update/cs/client/web references/localhost/UpdateCustomers.cs
//-----------------------------------------------------------------------
//  This file is part of the Microsoft .NET SDK Code Samples.
// 
//  Copyright (C) Microsoft Corporation.  All rights reserved.
// 
//This source code is intended only as a supplement to Microsoft
//Development Tools and/or on-line documentation.  See these other
//materials for detailed information regarding Microsoft code samples.
// 
//THIS CODE AND INFORMATION ARE PROVIDED AS IS WITHOUT WARRANTY OF ANY
//KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
//IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
//PARTICULAR PURPOSE.
//-----------------------------------------------------------------------

namespace Microsoft.Samples.Windows.Forms.Cs.Update.localhost {
    using System.Xml.Serialization;
    using System.Web.Services.Protocols;
    using System.Web.Services;
    
    
    [System.Web.Services.WebServiceBindingAttribute(Name="UpdateCustomersSoap", Namespace="http://tempuri.org/")]
    public class UpdateCustomers : System.Web.Services.Protocols.SoapHttpClientProtocol {
        
        public UpdateCustomers() {
            this.Url = "http://localhost/quickstart/winforms/samples/data/update/cs/UpdateCustomers.asmx";
        }
        
        [System.Web.Services.Protocols.SoapDocumentMethodAttribute("http://tempuri.org/Save", ParameterStyle=System.Web.Services.Protocols.SoapParameterStyle.Wrapped)]
        public System.Data.DataSet Save(System.Data.DataSet customersDataSet) {
            object[] results = this.Invoke("Save", new object[] {customersDataSet});
            return ((System.Data.DataSet)(results[0]));
        }
        
        public System.IAsyncResult BeginSave(System.Data.DataSet customersDataSet, System.AsyncCallback callback, object asyncState) {
            return this.BeginInvoke("Save", new object[] {customersDataSet}, callback, asyncState);
        }
        
        public System.Data.DataSet EndSave(System.IAsyncResult asyncResult) {
            object[] results = this.EndInvoke(asyncResult);
            return ((System.Data.DataSet)(results[0]));
        }
    }
}

File: vs2003_cd02/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/QuickStart/winforms/samples/data/update/vb/client/web references/localhost/UpdateCustomers.vb
'-----------------------------------------------------------------------
'  This file is part of the Microsoft .NET SDK Code Samples.
' 
'  Copyright (C) Microsoft Corporation.  All rights reserved.
' 
'This source code is intended only as a supplement to Microsoft
'Development Tools and/or on-line documentation.  See these other
'materials for detailed information regarding Microsoft code samples.
' 
'THIS CODE AND INFORMATION ARE PROVIDED AS IS WITHOUT WARRANTY OF ANY
'KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
'IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
'PARTICULAR PURPOSE.
'-----------------------------------------------------------------------

Imports System.Xml.Serialization
Imports System.Web.Services.Protocols
Imports System.Web.Services

Namespace Microsoft.Samples.Windows.Forms.Vb.Update.localhost

    <System.Web.Services.WebServiceBindingAttribute(Name:="UpdateCustomersSoap", [Namespace]:="http://tempuri.org/")> _
    Public Class UpdateCustomers
        Inherits System.Web.Services.Protocols.SoapHttpClientProtocol

        Public Sub New()
            MyBase.New()
            Me.Url = "http://localhost/quickstart/winforms/samples/data/update/vb/UpdateCustomers.asmx"
        End Sub

        <System.Web.Services.Protocols.SoapDocumentMethodAttribute("http://tempuri.org/Save", ParameterStyle:=System.Web.Services.Protocols.SoapParameterStyle.Wrapped)> _
        Public Function Save(ByVal customersDataSet As System.Data.DataSet) As System.Data.DataSet
            Dim results() As Object = Me.Invoke("Save", New Object() {customersDataSet})
            Return CType(results(0), System.Data.DataSet)
        End Function

        Public Function BeginSave(ByVal customersDataSet As System.Data.DataSet, ByVal callback As System.AsyncCallback, ByVal asyncState As Object) As System.IAsyncResult
            Return Me.BeginInvoke("Save", New Object() {customersDataSet}, callback, asyncState)
        End Function

        Public Function EndSave(ByVal asyncResult As System.IAsyncResult) As System.Data.DataSet
            Dim results() As Object = Me.EndInvoke(asyncResult)
            Return CType(results(0), System.Data.DataSet)
        End Function
    End Class
End Namespace
